stories/Color.stories.js

```diff
@@ -55,6 +55,6 @@ storiesOf("Color", module)
     <ColorStory
       title="Alternate Colors"
       description="The alternate colors are rounding out the set of colors we have in the system, but will likely not be used that often. A common use case for these colors will be data visualizations."
-      array={["orange", "yellow", "purple", "blilet"]}
+      array={["orange", "yellow", "purple", "blilet", "midnight"]}
```

Comment:
@justinanastos my only question is where this color should really live? I just stuck it here for now, but considering it's specifically for a UI theme...does it belong elsewhere?

Reply:
Nope! I have no preference here. I'd like to revamp this eventually but I have no idea how to do that. This looks good to me.